packages/gatsby-mdx/mdx-renderer.js

```diff
@@ -0,0 +1,15 @@
+export default ({ scope = {}, components = {}, children, ...props }) => {
```

Comment:
the MDX rendering runtime. Allows you to pass in props to be used on the component.